examples/tls/tls_common.h

```diff
@@ -311,6 +322,240 @@ static inline void CloseAndCleanupSocket(SockIoCbCtx* sockIoCtx)
         sockIoCtx->listenFd = -1;
     }
 }
+#else /* USE_WOLFSSL_IO */
+
+/* Copied from wolfSSL server.c */
+static WC_INLINE int wolfIO_LastError(void)
+{
+#ifdef USE_WINDOWS_API
+    return WSAGetLastError();
+#elif defined(EBSNET)
+    return xn_getlasterror();
+#else
+    return errno;
+#endif
+}
+
+static inline int SockIORecv(WOLFSSL* ssl, char* buff, int sz, void* ctx)
+{
+    SockIoCbCtx* sockCtx = (SockIoCbCtx*)ctx;
+    int recvd;
+    int err;
+
+    (void)ssl;
+
+    /* Receive message from socket */
+    if ((recvd = wolfIO_Send(sockCtx->fd, buff, sz, 0)) == -1) {
```

Comment:
As discussed this is wrong, should be `wolfIO_Recv`, however you'll be refactoring away the requirement to rely on wolfIO here.